hack/install-requirements.sh

```diff
@@ -22,7 +22,7 @@ echo "> [DEPRECATED] Installing requirements"
 
 export GO111MODULE=on
 curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.41.1
-curl -s "https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3" | bash -s -- --version 'v3.5.4'
+curl -s "https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3" | bash -s -- --version 'v3.6.3'
```

Comment:
I think the helm version is maintained in `hack/tools.mk` as well (from my understanding the reason for this is that this script is deprecated and hack/tools.mk is a new way of dealing with such dependencies).

```diff
--- a/hack/tools.mk
+++ b/hack/tools.mk
@@ -49,7 +49,7 @@ YQ                         := $(TOOLS_BIN_DIR)/yq
 # default tool versions
 DOCFORGE_VERSION ?= v0.21.0
 GOLANGCI_LINT_VERSION ?= v1.44.0
-HELM_VERSION ?= v3.5.4
+HELM_VERSION ?= v3.6.3
 KIND_VERSION ?= v0.11.1
 SKAFFOLD_VERSION ?= v1.35.0
 YQ_VERSION ?= v4.9.6
```

Reply:
Thanks @ialidzhikov, this is super helpful and will correct. Are you familiar with any criteria for version selection? `3.8.0` was recently released, that strikes me as too new, I selected `3.6.3` as the oldest feature series that supports M1 to disturb the least. 

Is the latest `3.7.x` a better choice?

Reply:
I am not very familiar with the upstream changes but `v3.6.3` seems safer for the purpose of this PR. If needed, in the future we can upgrade to higher versions.